go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/container-storage-interface/spec v1.7.0
 	github.com/coredns/corefile-migration v1.0.18
 	github.com/coreos/go-oidc v2.1.0+incompatible
-	github.com/coreos/go-systemd/v22 v22.3.2
+	github.com/coreos/go-systemd/v22 v22.4.0
```

Comment:
probably worth a separate PR to get the dep bump in, this will have the highest rebase churn